test/extended/testdata/deployments/custom-deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         echo Halfway
         openshift-deploy
         echo Finished
+        sleep 1
```

Comment:
Without that sleep, the line before never gets to the caller in [here](https://github.com/ncdc/origin/blob/1.5-rebase/test/extended/deployments/deployments.go#L418-L425).